config/locales/actions.en.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  actions:
+    archive_prompt:
+      label: "Archive"
+      confirmation: "Are you sure you want to archive %{name}?"
```

Comment:
that won't work for some cases.

Let's say you have an object whose identifier is a date

> Are you sure you want to archive "this claim from the 19th of May"

vs

> Are you sure you want to archive this claim from the 19th of May

Agreed?